src/main/resources/application-prod-gcp-k9saksbehandling.properties

```diff
@@ -1,2 +1,5 @@
 context.path=/k9/inntektsmelding
 inntektsmelding.skjema.lenke=https://arbeidsgiver.nav.no/k9-im-dialog
+
+aareg.rs.url=http://aareg-services-nais.arbeidsforhold/api/v1/arbeidstaker
```

Comment:
Er det ikke litt rart at det er samme url her i test og i prod?

Reply:
Dette er et service discovery URL som kubernetes klarer å finne ut av, men siden appen hetter det samme i både dev og prod så er det nok ok.

Urlen består av `http://<appnavn i kubernetes>.<namespace i kubernetes>/<path til endepunkt>`

Reply:
Aha! Takk for forklaring